mousestyles/dynamics/__init__.py

```diff
@@ -337,3 +337,92 @@ def mcmc_simulation(mat_list, n_per_int):
             simu_result[index] = int(state_j)
     simu_result = np.array(simu_result, dtype=int)
     return simu_result
+
+
+def get_score(true_day, simulated_day, weight=[1, 10, 50, 1]):
```

Comment:
It's the same. The weights are the numbers on the nominator, and they are chosen based on the inverse of the proportion of this status. For example, because only 2 percent of one day is in the drink status, when we get drink correct, we should have a much higher reward to be 50 in the score; only 10 percent of one day is in the eat status, so when we get eat correct, we should have a higher reward to be 10.